src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlClientEventSource.Windows.cs

```diff
@@ -35,7 +35,7 @@ protected override void OnEventCommand(EventCommandEventArgs e)
             // If registration fails, all write and unregister commands will be a no-op.
 
             // If managed networking is enabled, don't call native wrapper methods
-#if netcoreapp
+#if NETCOREAPP
             if (AppContext.TryGetSwitch("Switch.Microsoft.Data.SqlClient.UseManagedNetworkingOnWindows", out bool isEnabled) && isEnabled)
```

Comment:
Looking carefully at this code right now, I think this `#if` block must also check for .NET Standard?